README.md

```diff
@@ -34,14 +34,16 @@ Alternatively, you can clone the snap-box repository [using this GitHub template
 
 ## Setup
 
-You will need an Infura API key to fork the Ethereum blockchain in your local instance of Ganache. Copy the `.env.dist` file in `packages/truffle/` to a new file `.env` and update the `INFURA_PROJECT_ID` variable with your API key.
+You can add your own Infura API key to fork the Ethereum blockchain in your local instance of Ganache. Copy the `.env.dist` file in `packages/truffle/` to a new file `.env` and update the `INFURA_PROJECT_ID` variable with your API key.
 
 Then, you can take your Secret Recovery Phrase from MetaMask Flask and put it as the `MNEMONIC_PHRASE` in this same file. This will make your first Ethereum account as the deployer of the test contracts in Ganache and provide you with a balance for the first 10 accounts for local testing. _Remember, you should never share your Secret Recovery Phrase with anyone, and you should never use a Secret Recovery Phrase from any version of MetaMask that you use to custody real funds. This `.env` file is never and should never be uploaded to GitHub (it is explicitly excluded in `.gitignore`). This is only for testing locally with Ganache._
 
+Alternatively, you can take the Mnemonic Phrase generated by Truffle and use that when setting up MetaMask Flask. This will guarantee that your transaction history is empty each time you start testing.
```

Comment:
```suggestion
Alternatively, you can take the Secret Recovery Phrase generated by Truffle and use that when setting up MetaMask Flask. This will guarantee that your transaction history is empty each time you start testing.
```